bootstrapvz/providers/ec2/tasks/ami.py

```diff
@@ -125,3 +125,10 @@ def run(cls, info):
             registration_params['sriov_net_support'] = 'simple'
 
         info._ec2['image'] = info._ec2['connection'].register_image(**registration_params)
+
+        # Setting up tags on the AMI
+        if info.manifest.tags:
```

Comment:
Just for completeness sake you should probably check for `is not None` here, this way it's possible to know exactly which type the variable is in the negative case. Alternatively, you could make it an empty `dict` during init and check with `len() > 0` instead.  

Also: Why the `else: pass`?